content/soda-gettingstarted/quickstart.md

```diff
@@ -25,72 +26,56 @@ To have more control of the installation configurations, or try out various conf
 	|Greenland        | v1.1.0         |
 	|Faroe            | v1.0.0         |
 	|Elba             | v0.20.0        |
-	Use the **RELEASE TAG** in the commands below wherever mentioned RELEASE-TAG.
 
-## Quick Installation and experience of SODA releases
 
-{{% notice info %}}
-**Ubuntu 16.04 support for SODA is deprecated as of SODA Jerba release v1.4.0.** <br />
-**If you have a requirement to install SODA on Ubuntu 16.04, please contact us on slack and we will try to help with the setup.**
-{{% /notice %}}
+## Quick Installation and experience of SODA releases
 
-This procedure applies to 18.04 Linux distributions.
+This procedure applies to 18.04 & 20.04 Linux distributions.
 
 - **Prerequisite**
- 
-   - Get superuser priviledges:
-    
-      `sudo -s`
-    
+
     - Install Basic Dependencies:
       
-      `apt-get update && apt-get install -y git make curl wget libltdl7 libseccomp2 libffi-dev gawk`
-	    
-     - Install Docker:
-	
-       - Ubuntu 18.04:
-
-       `wget https://download.docker.com/linux/ubuntu/dists/bionic/pool/stable/amd64/docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb && dpkg -i docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb`
-    
-    - Install docker-compose:
-
-	    -   curl -L https://github.com/docker/compose/releases/download/1.23.1/docker-compose-$(uname -s)-$(uname -m) -o /usr/local/bin/docker-compose
-		-   chmod +x /usr/local/bin/docker-compose
-    -   Required golang version: 1.13.x
-	    -   Run following command `go version` and check the version
-	    -   If the required version is not installed on the system, install golang,1.13.9 using:
-		   - wget **[https://storage.googleapis.com/golang/go1.13.9.linux-amd64.tar.gz](https://storage.googleapis.com/golang/go1.13.9.linux-amd64.tar.gz)**
-		- tar -C /usr/local -xzf go1.13.9.linux-amd64.tar.gz
-		- echo export PATH=$PATH:/usr/local/go/bin >> /etc/profile
-		- echo export GOPATH=$HOME/gopath >> /etc/profile
-		- source /etc/profile
+      `sudo apt-get update && sudo apt-get install -y git`
 
 	- **Get and Install Release Binaries**
 	
-	{{% notice info %}}
 
-	Please replace RELEASE_TAG with the corresponding value from Release Tag table above (ex: v1.6.0 etc)
 
-	{{% /notice %}}
+**_NOTE:_**  The note content.Please replace RELEASE_TAG with the corresponding value from Release Tag table above (ex: v1.8.0 etc)
+
+
+```bash
+wget  [https://github.com/sodafoundation/installer/releases/download/<RELEASE_TAG>/installer-[RELEASE_TAG].tar.gz](https://github.com/sodafoundation/installer/releases/download/[RELEASE_TAG]/installer-[RELEASE_TAG].tar.gz)
```

Comment:
Change the Markdown URL format to simple URL [https://github.com/sodafoundation/installer/releases/download/<RELEASE_TAG>/installer-[RELEASE_TAG].tar.gz](https://github.com/sodafoundation/installer/releases/download/[RELEASE_TAG]/installer-[RELEASE_TAG].tar.gz) - > https://github.com/sodafoundation/installer/releases/download/[RELEASE_TAG]/installer-[RELEASE_TAG].tar.gz